homeassistant/components/ephember/climate.py

```diff
@@ -89,8 +89,9 @@ def __init__(self, ember, zone):
         self._ember = ember
         self._zone_name = zone_name(zone)
         self._zone = zone
-        self._hot_water = zone_is_hot_water(zone)
-
+        self._hot_water = zone['deviceType'] == 4
+        """4 is a specific device type for immersions returned by EPH. Hot Water temp cannot be changed"""
```

Comment:
This should be a new constant in `ephember/const.py`

Reply:
I can take a look at this one @roberty99

Reply:
Should be addressed here: https://github.com/roberty99/core/pull/2